index.d.ts

```diff
@@ -138,7 +138,7 @@ declare module "hyperapp" {
     readonly tag: Tag<S>
     readonly key: Key
     memo?: PropList<S, C>
-    events?: Record<string, Action<S>>
+    events?: Record<string, Reaction<S>>
```

Comment:
😅

Reply:
`Reaction` has now been inlined and removed.